.travis.yml

```diff
@@ -1,7 +1,7 @@
-sudo: false
+sudo: required
 language: scala
 scala:
-- 2.11.7
+- 2.11.11
```

Comment:
Should we make this 2.12.2?  I think we want the primary build to be 2.12 with 2.11 supported by cross-build.